### DISCUSSION TITLE
Add Apple TV as Speaker to Music Assistant (HA)

### DISCUSSION THREAD

**Discussion by JohnJay77** [music-assistant/support#3317] [Feature requests and ideas]
Hello
When I installed the Music Assistant in my home theater, I wanted to add my players/speakers.
It found my 3 Sonos players and my 2 HomePods straight away. Unfortunately, it didn’t find my 2 Apple TV 4k.
Does anyone know how I can add the 2 Apple TVs too?
Thank you very much

**Comment by OzGav** [comment]
Duplicate. Search before posting https://github.com/orgs/music-assistant/discussions/2528
